### PR TITLE
chore: pretty print the effective client settings

### PR DESCRIPTION
**Pull request by kongweihan**

### CODE REVIEW

**Comment by kongweihan** [comment]
@igorbernstein2

File: bigtable-client-core-parent/bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java [hunk 1]
```diff
@@ -353,4 +352,74 @@ public Set<TableName> getDisabledTables() {
   public BigtableApi getBigtableApi() {
     return bigtableApi;
   }
+
+  // A helper function to parse the pretty print the string from ToStringHelper.
+  public static class ToStringHelperPrettyPrinter {
```
Comment (mutianf) [inline bigtable-client-core-parent/bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java:357]: I don't think this needs to be public

File: bigtable-client-core-parent/bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java [hunk 2]
```diff
@@ -353,4 +352,74 @@ public Set<TableName> getDisabledTables() {
...
+      IN_SET,  // Print verbatim for sets, including commas and spaces, e.g. "[foo, bar]".
+    }
+
+    private static void appendIndent(StringBuilder sb, int level) {
+      for (int i = 0; i < level; i++) {
+        sb.append("  ");
+      }
+    }
+
+    public static String print(String str) {
```
Comment (mutianf) [inline bigtable-client-core-parent/bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java:371]: Same here, I don't think this needs to be public

**Review by mutianf — CHANGES_REQUESTED** [review]
Can you also fix the pr name to follow the convention? something like `chore: pretty print the settings`

**Review by mutianf — APPROVED** [review]

**Pull request merged**